recipe/laravel.php

```diff
@@ -35,6 +35,16 @@
     'storage/logs',
 ]);
 
+set('laravel_version', function () {
+    $result = run('{{bin/php}} {{release_path}}/artisan --version');
+
+    preg_match_all('/version\ (.+?)$/', $result, $matches);
```

Comment:
Laravel 4: `Laravel Framework version 4.2.22`
Laravel 5.1: `Laravel Framework version 5.1.35 (LTS)`
Laravel 5.2: `Laravel Framework version 5.2.45`
Laravel 5.3: `Laravel Framework version 5.3.30`
Laravel 5.4: `Laravel Framework 5.4.22`
Lumen: `Laravel Framework version Lumen (5.1.7) (Laravel Components 5.1.*)`

Something like this? `([0-9.]+)`
http://regexr.com/3g2em